### DISCUSSION TITLE
表单增加”学校的历史热搜“建议

### DISCUSSION THREAD

**Discussion by nexmoe** [CollegesChat/university-information#86] [Enhancement]
1. 更全面了解学校历史
2. 比其他表单内容更具有客观性
3. 真实性可验证（互联网是有记忆的

可以获取热搜的渠道

- 微博
- 知乎
- 哔哩哔哩
- 各种新闻

**Comment by Richard-Zheng** [comment]
获取微博历史热搜的接口：https://www.zhihu.com/question/38155682
